### PR TITLE
Make codegen_test more self contained

### PR DESCRIPTION
**Pull request by chalin**
Addressing the codegen_test `TODO(jmesserly): it'd be nice to do all cleanup here …`.

### CODE REVIEW

**Comment by jmesserly** [comment]
nice, thank you! this also fixes #99 :)

File: tool/travis.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 set -e
 
 function clean {
```
Comment (kevmoo) [inline tool/travis.sh]: @jmesserly I think you should be able to kill the `clean` function entirely from the travis script. Could you try?

Comment (jmesserly) [inline tool/travis.sh]: sure thing

Comment (kevmoo) [inline tool/travis.sh]: Actually, I already took care of it in latest. We good. :smile:

Comment (jmesserly) [inline tool/travis.sh]: haha. yeah I went to edit it and was like "where did clean go?!" :)

**Pull request merged**